src/audio/CMakeLists.txt

```diff
@@ -124,7 +124,7 @@ set(sof_audio_modules volume src asrc eq-fir eq-iir dcblock crossover)
 set(volume_sources volume/volume.c volume/volume_generic.c)
 set(src_sources src/src.c src/src_generic.c)
 set(asrc_sources asrc/asrc.c asrc/asrc_farrow.c asrc/asrc_farrow_generic.c)
-set(eq-fir_sources eq_fir/eq_fir.c eq_fir/fir.c)
+set(eq-fir_sources eq_fir/eq_fir.c)
```

Comment:
I still need to fix the KConfig and cmake files to control FIR library build. This commit only does the basic move of FIR functions out from EQ for generic usage.

Reply:
@singalsu any ETA for the build file changes ?

Reply:
I'll try to complete today the build related. I'll also do the change to use common model set/get functions.

Reply:
This is resolved.